### ISSUE TITLE
Filter emoji

### ISSUE THREAD

**Issue by AgustinTiseira** [ealush/emoji-picker-react#274]
Hi everybody, i have a doubt. How can I filter the emojis displayed?
it would be useful to show only some categories similar to the "categories" attribute but also to be able to limit the emojis shown from that category.
for example: 

`<EmojiPicker categories={[
{
    category: 'smileys_people',
    name: 'Faces...',
    filterEmoji: ["Smile", "Grin", "Heart Eyes"]
  }
]} />`

please if there is already a way to do this let me know. Thank you!

**Comment by ealush** [comment]
Hey @AgustinTiseira 👋 thanks for opening this issue. That's a pretty useful suggestion. Let me iterate on it and see what I come up with.

**Comment by ealush** [comment]
Sorry for taking some time. I've been traveling on vacation this past month (and still am!) so only had little availability to play with this, but here's an idea:

Instead of adding this property that's specific to your use case, instead I can add a predicate function to filter the emojis based on any criteria, their names, their unified code, etc... 

Here's the PR with an example: https://github.com/ealush/emoji-picker-react/pull/282

Please let me know if that solves it for you.

Published as an RC version on:
```emoji-picker-react@4.4.0-rc.1```

**Comment by SweetSoul** [comment]
Hey @ealush that works for me, I was having the same problem! It would be awesome to have this merged

**Comment by ricjohn-acosta** [comment]
Wondering if you have a temporary solution for this @SweetSoul?

**Comment by bennetmi** [comment]
@ealush was wondering if there was any update on this? Have enjoyed using this awesome menu, but would need to remove the gun and middle finger emojis to continue doing so. Would this make it possible?

**Comment by cwikio** [comment]
yeah, is that done? can we filter out emojis? Many thanks in advance.

**Comment by ealush** [comment]
This is in the package for the past 3 years. closing
